service/src/main/kotlin/fi/espoo/evaka/application/PendingDecisionEmailService.kt

```diff
@@ -29,7 +29,9 @@ class PendingDecisionEmailService(
         asyncJobRunner.sendPendingDecisionEmail = ::doSendPendingDecisionsEmail
     }
 
-    val fromAddress = env.getRequiredProperty("mail_reply_to_address")
+    val senderAddress: String = env.getRequiredProperty("mail_reply_to_address")
+    val senderName: String = env.getRequiredProperty("mail_sender_name")
```

Comment:
As these props aren't set in evaka-infra, would you mind fixing them to follow the fi.espoo.... format in this PR?